base/CHANGELOG.md

```diff
@@ -75,3 +75,9 @@
 | cypress/base:16.14.2 | 16.14.2 | Debian | [/16.14.2](16.14.2) | `🚫` | `🚫` | `🚫` |
 | cypress/base:16.14.2-slim | 16.14.2 | Debian | [/16.14.2](16.14.2) | `🚫` | `🚫` | `🚫` |
 | cypress/base:14.17.6-slim | 14.17.6 | Debian | [/14.17.6](14.17.6) | `🚫` | `🚫` | `🚫` |
+
+## Notes
```

Comment:
won't the generate-changelog automation obliterate this?

Reply:
It's still in the automation, so I'm not sure why it was removed in the first place.